WordPress/Classes/Extensions/Colors and Styles/WPStyleGuide+ApplicationStyles.swift

```diff
@@ -108,6 +108,16 @@ extension WPStyleGuide {
                                                    for: .disabled)
 
     }
+
+    class func configureToolbarAppearance() {
+        if #available(iOS 15.0, *) {
```

Comment:
At this point I'm still unsure why on iOS 15 the toolbar is entirely translucent unless we explicitly set the appearance somewhere. I added this as a fallback for all toolbars, in case this happens in other areas.